.github/workflows/integration.yml

```diff
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        redis-stack-version: ["6.2.6-v7", "7.0.6-RC8", "edge"]
+        redis-stack-version: ["6.2.6-v9", "7.2.0-RC3", "edge"]
```

Comment:
Does this mean these versions would required to be continuously updated with every new Redis (Stack) release?

Reply:
@chayim

Reply:
@chayim @shacharPash - There can be slight breaking changes between RC and GA versions [it's happened!](https://github.com/StackExchange/StackExchange.Redis/pull/2454#issuecomment-1559973204) - This is fine while this is pre-1.0, but after that if you are going to point at these you might consider flagging pre-GA APIs as not covered under semver (in case there's a need to break them in the future), that way you don't need to push a major version every time there's a minor compatibility break between a pre-GA and GA API.

Just something to consider.

Reply:
Much like redis - of course.